com.unity.toonshader/.buginfo

```diff
@@ -0,0 +1 @@
+package: Unity Toon Shader
```

Comment:
This document says some information is required.
> system: jira
> server: jira.unity3d.com
> issuetype: Bug
> project: ABC ← (The project key where issues found in the associated code/tests should be logged)
> area: AreaName OR package: PackageName ← (pick only one based on what the code should be associated with; a list of available areas can be found here and a list of available packages can be found here)


https://internaldocs.unity.com/editor_and_runtime_development_guide/DevelopmentProcess/authoring-changes/code-ownership/#required-information-for-a-buginfo-file

Reply:
Fixed. Thank you.